### PR TITLE
Review thomas

### PR DESCRIPTION
**Pull request by Thomas-Smets**
Some small fixes for startLiquidation in AccountV1.sol

### CODE REVIEW

File: test/fuzz/accounts/AccountV1/AuctionBid.fuzz.t.sol [hunk 1]
```diff
@@ -9,7 +9,7 @@ import { Constants, AccountV1_Fuzz_Test } from "./_AccountV1.fuzz.t.sol";
 import { AccountExtension, AccountV1 } from "../../../utils/Extensions.sol";
 
 /**
- * @notice Fuzz tests for the "auctionBuy" of contract "AccountV1".
+ * @notice Fuzz tests for the "auctionBid" of contract "AccountV1".
  */
 contract AuctionBuy_AccountV1_Fuzz_Test is AccountV1_Fuzz_Test {
```
Comment (j-vp) [inline test/fuzz/accounts/AccountV1/AuctionBid.fuzz.t.sol:14]:
```suggestion
contract AuctionBid_AccountV1_Fuzz_Test is AccountV1_Fuzz_Test {
```

File: test/fuzz/accounts/AccountV1/AuctionBoughtIn.fuzz.t.sol [hunk 2]
```diff
@@ -9,7 +9,7 @@ import { Constants, AccountV1_Fuzz_Test } from "./_AccountV1.fuzz.t.sol";
 import { AccountExtension, AccountV1 } from "../../../utils/Extensions.sol";
 
 /**
- * @notice Fuzz tests for the "auctionBuy" of contract "AccountV1".
+ * @notice Fuzz tests for the "auctionBid" of contract "AccountV1".
```
Comment (j-vp) [inline test/fuzz/accounts/AccountV1/AuctionBoughtIn.fuzz.t.sol:12]:
```suggestion
 * @notice Fuzz tests for the "auctionBoughtIn" of contract "AccountV1".
```

**Review by zekiblue — APPROVED** [review]
gtg

File: test/fuzz/accounts/AccountV1/AuctionBoughtIn.fuzz.t.sol [hunk 3]
```diff
@@ -9,7 +9,7 @@ import { Constants, AccountV1_Fuzz_Test } from "./_AccountV1.fuzz.t.sol";
...
  */
 contract AuctionBuyIn_AccountV1_Fuzz_Test is AccountV1_Fuzz_Test {
```
Comment (j-vp) [inline test/fuzz/accounts/AccountV1/AuctionBoughtIn.fuzz.t.sol:14]:
```suggestion
contract AuctionBoughtIn_AccountV1_Fuzz_Test is AccountV1_Fuzz_Test {
```

**Review by j-vp — APPROVED** [review]
gtg

**Pull request merged**